test/extension/config.test.ts

```diff
@@ -1,29 +1,32 @@
 import { expect } from "chai";
+import sinon from "sinon";
 import * as vscode from "vscode";
 
-import {Config} from "../../src/extension/config";
-
-// Original functions
-const createTextEditorDecorationType = vscode.window.createTextEditorDecorationType;
-const executeCommand = vscode.commands.executeCommand;
-const getConfiguration = vscode.workspace.getConfiguration;
+import { Config } from "../../src/extension/config";
 
 let showGutterCoverage: boolean;
 let iconPathDark: string;
 let iconPathLight: string;
 
-suite("Config Tests", function() {
-    const fakeVscode: any = {
-        createTextEditorDecorationType: (options) => {
-            expect(Object.keys(options)).to.have.lengthOf(4);
-            return {};
+suite("Config Tests", () => {
+    const fakeContext: any = {
+        asAbsolutePath: () => {
+            return "123";
         },
+    };
 
-        executeCommand: () => {
-            return ;
-        },
+    let stubCreateTextEditorDecorationType: sinon.SinonStub;
+    const fakeCreateTextEditorDecorationType = (options: vscode.DecorationRenderOptions) => {
+        expect(Object.keys(options)).to.have.lengthOf(4);
+        return {} as vscode.TextEditorDecorationType;
+    };
 
-        getConfiguration: () => {
+    setup(() => {
+        stubCreateTextEditorDecorationType = sinon.stub(vscode.window, "createTextEditorDecorationType");
```

Comment:
[F] This seemed necessary as we want to change the implementation for some tests